src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.in.json

```diff
@@ -15,6 +15,18 @@
       ],
       "platforms": [ "win-x64", "linux-x64", "osx-x64", "osx-arm64" ],
       "extends" : [ "microsoft-net-runtime-android", "microsoft-net-runtime-android-aot" ]
+    },
+    "android-33": {
+      "description": "Preview support for Android API-33.",
+      "packs": [
+        "Microsoft.Android.Ref.33",
```

Comment:
@jonathanpeppers: can we/should we insert "Preview" into these package names?  Is that possible?